src/Core/Layer/Layer.js

```diff
@@ -92,6 +92,7 @@ GeometryLayer.prototype.attach = function attach(layer) {
 GeometryLayer.prototype.detach = function detach(layer) {
     const count = this._attachedLayers.length;
     this._attachedLayers = this._attachedLayers.filter(attached => attached.id != layer.id);
+    layer.parentLayer = null;
```

Comment:
It seems to me that we have avoided adding a parent layer.
This isn't detailed and motivated in the PR and in the commit.
If you want to do that, you might have to be a separate PR

Reply:
No, it is currently present in `View`. So it is ok imho.

Reply:
I meant to keep the link with the parent

Reply:
I preferred to add it where I needed it, than just opening a PR without precise application case (as it is a minor change really). That being said, if you agree with this `layer.parentLayer`, I can open a new PR with it, no problem.